### PR TITLE
Fix svg export

### LINKED ISSUE #110: Exported connections are missing the arrow-head
## Describe the bug
The arrow head seems to be missing in exported svg.

## Screenshot

Create_Elements

grafik

### PR DESCRIPTION
**Pull request by 0x4d**
Closes #110 

## Describe your changes

- Added a properties with references to defs to the Canvas
- Refactored Exporter.getGraphics() to return a complete and valid SVG

### CODE REVIEW

File: src/core/Canvas.js [hunk 1]
```diff
@@ -11,17 +11,25 @@ export class Canvas extends BaseCanvas {
 
     constructor (config, eventBus, graphicsFactory, elementRegistry) {
         super(config, eventBus, graphicsFactory, elementRegistry);
-        this.createUiGroup();
+        this._uiGroup = this.createUiGroup();
```
Comment (tim-kilian) [inline src/core/Canvas.js:14]: Why the underscore?

File: src/draw/Renderer.js [hunk 2]
```diff
@@ -36,7 +29,7 @@ export class Renderer extends BaseRenderer {
                 markerHeight: 10,
                 orient: 'auto',
             });
-            append(defs, marker);
+            append(this.canvas._defs, marker);
```
Comment (tim-kilian) [inline src/draw/Renderer.js:32]: looks nice

**Review by tim-kilian — APPROVED** [review]

**Comment by tim-kilian** [comment]
Published https://www.npmjs.com/package/renew-lib/v/1.0.1

File: src/core/Canvas.js (hunk 1 above)
Comment (0x4d) [reply src/core/Canvas.js:14]: Mainly to be consistent with the parent class.

**Pull request merged**